package.json

```diff
@@ -39,7 +39,7 @@
     "locator-handlebars": "https://github.com/zillow/locator-handlebars/tarball/master",
     "locator-yui": "https://github.com/zillow/locator-yui/tarball/master",
     "nopt": "^3.0.1",
-    "yui": "3.15.0"
+    "yui": "3.18.1"
```

Comment:
Probably should bump the zlocator version too to a major version change to indicate incompatibility.